packages/teleterm/src/ui/DocumentCluster/ClusterResources/Servers/Servers.tsx

```diff
@@ -35,44 +32,89 @@ export default function Container() {
   return <ServerList {...state} />;
 }
 
+function getEmptyTableText(status: AttemptStatus) {
+  switch (status) {
+    case 'error':
+      return 'Failed to fetch servers.';
+    case '':
+      return 'Searching…';
+    case 'processing':
+      return 'Searching…';
+    case 'success':
+      return 'No servers found.';
+  }
+}
+
 function ServerList(props: State) {
-  const { servers = [], getSshLogins, connect, syncStatus } = props;
+  const {
+    getSshLogins,
+    connect,
+    fetchAttempt,
+    agentFilter,
+    pageCount,
+    customSort,
+    prevPage,
+    nextPage,
+    updateQuery,
+    onAgentLabelClick,
+    updateSearch,
+  } = props;
+  const servers = fetchAttempt.data?.agentsList.map(makeServer) || [];
+  const disabled = fetchAttempt.status === 'processing';
+  const emptyTableText = getEmptyTableText(fetchAttempt.status);
+
   return (
     <>
-      {syncStatus.status === 'failed' && (
-        <Danger>{syncStatus.statusText}</Danger>
+      {fetchAttempt.status === 'error' && (
+        <Danger>{fetchAttempt.statusText}</Danger>
       )}
-      <Table
-        columns={[
-          {
-            key: 'hostname',
-            headerText: 'Hostname',
-            isSortable: true,
-          },
-          {
-            key: 'addr',
-            headerText: 'Address',
-            isSortable: true,
-            render: renderAddressCell,
-          },
-          {
-            key: 'labelsList',
-            headerText: 'Labels',
-            render: renderLabelCell,
-          },
-          {
-            altKey: 'connect-btn',
-            render: server =>
-              renderConnectCell(
-                () => getSshLogins(server.uri),
-                login => connect(server.uri, login)
-              ),
-          },
-        ]}
-        emptyText="No Nodes Found"
-        data={servers}
-        pagination={{ pageSize: 15, pagerPosition: 'bottom' }}
+      <SearchPanel
+        updateQuery={updateQuery}
+        updateSearch={updateSearch}
+        pageCount={pageCount}
+        filter={agentFilter}
+        showSearchBar={true}
+        disableSearch={disabled}
       />
+      <DarkenWhileDisabled disabled={disabled}>
+        <Table
+          columns={[
+            {
+              key: 'hostname',
+              headerText: 'Hostname',
+              isSortable: true,
+            },
+            {
+              key: 'addr',
+              headerText: 'Address',
+              isSortable: false,
+              render: renderAddressCell,
```

Comment:
Idk what has changed exactly, but the address column breaks up that arrow next to tunnel now.

![address column comparison](https://user-images.githubusercontent.com/27113/203342643-5dfdafd2-2512-435f-9fe2-d5ba74e77d33.png)

Reply:
I don't see the same thing on the current branch. 
<img width="558" alt="Screenshot 2022-11-23 at 3 22 21 PM" src="https://user-images.githubusercontent.com/5201977/203483753-272b0d85-ec99-4fdb-93ef-46362ddd40cb.png">